hector_mapping/src/HectorMappingRos.cpp

```diff
@@ -231,106 +231,100 @@ HectorMappingRos::~HectorMappingRos()
     delete map__publish_thread_;
 }
 
-void HectorMappingRos::scanCallback(const sensor_msgs::LaserScan& scan)
-{
+void HectorMappingRos::scanCallback(const sensor_msgs::LaserScan& scan) {
```

Comment:
If you want, I can move back on the curly bracket changes, but this is how Google recommends it